### ISSUE TITLE
QPopupProxy - no-parent-event not working

### ISSUE THREAD

**Issue by nicholaszuccarelli** [quasarframework/quasar#4953]
I've got a problem using QPopupProxy not handling the no-parent-event prop.

I need my popup-proxy (in menu-mode) to use the QMenu no-parent-event property but it is not triggering at all.

If I explicitly do: (for example):
```
      <component
        :is="windowWidth <= breakpoint ? 'q-dialog' : 'q-menu'"
        no-parent-event
```

to manually switch the component depending on the window width, then I get my expected behaviour.

**Comment by pdanpdan** [comment]
QPopupProxy in QMenu mode should force `noParentEvent` prop to true. What are you trying to do that is not working?

**Comment by nicholaszuccarelli** [comment]
Ok so in my case, I have attached the PopupProxy to a QInput.
The typical behaviour is, when you click on the input it will toggle the QPopupProxy (open/close).

By adding noParentEvent prop, it will stop it from doing this behaviour (has to be handled manually), however when I use QPopupProxy with noParentEvent prop, it doesn’t stop the toggle behaviour.

If I instead used a QMenu, the behaviour works as intended.

**Comment by pdanpdan** [comment]
For me it looks like it's doing the right thing: https://codepen.io/pdanpdan/pen/wvwrXdJ?editors=1010

**Comment by nicholaszuccarelli** [comment]
The bug must've been fixed in the 1.1.0 release because I just tried in my app and I can't simulate the bug anymore.

Thank you :)
